rest-api/app_base.yaml

```diff
@@ -8,24 +8,8 @@ api_version: 1
 
 builtins:
 - deferred: on
+- appstats: on
```

Comment:
Shouldn't this be in app_nonprod.yaml too?

Reply:
no, the files still get concatenated.

Reply:
okay, after reading below comment, I moved builtins to the environment files and removed appstats from prod.